README.md

```diff
@@ -19,8 +19,8 @@ Minikube is a tool that makes it easy to run Kubernetes locally. Minikube runs a
 
 ### Requirements
 
-* [VirtualBox](https://www.virtualbox.org/wiki/Downloads), [VMware Fusion](https://www.vmware.com/products/fusion)
-or [KVM](http://www.linux-kvm.org/) installation
+* [VirtualBox](https://www.virtualbox.org/wiki/Downloads), [VMware Fusion](https://www.vmware.com/products/fusion),
+ [KVM](http://www.linux-kvm.org/) or [xhyve](https://github.com/mist64/xhyve) installation
```

Comment:
I got this running, and I don't have xhyve itself installed as far as I can tell. Maybe clarify that you just need the driver, not xhyve itself?